packages/orbit-components/src/primitives/ButtonPrimitive/components/ButtonPrimitiveContentChildren.js

```diff
@@ -2,13 +2,14 @@
 import * as React from "react";
 import styled from "styled-components";
 
+import { left } from "../../../utils/rtl";
 import defaultTheme from "../../../defaultTheme";
 import type { Props } from "./ButtonPrimitiveContentChildren";
 
 const StyledButtonPrimitiveContentChildren = styled.div`
   display: inline-block;
   width: ${({ contentWidth }) => contentWidth};
-  text-align: ${({ hasIcon }) => hasIcon && "left"};
+  text-align: ${({ hasIcon }) => hasIcon && left};
```

Comment:
I'm pretty sure this is the right thing to do. The text inside button components with `fullWidth` is currently always aligned to the left, but I assume it should be aligned to the right? This will have an impact on all button components, so it's an important decision, but the Wizard component didn't look right without it.

Before:

<img width="531" alt="Screen Shot 2020-10-15 at 12 53 18" src="https://user-images.githubusercontent.com/471278/96114073-67ff0d00-0ee5-11eb-86f1-b6aa60076615.png">

After:

<img width="531" alt="Screen Shot 2020-10-15 at 12 52 27" src="https://user-images.githubusercontent.com/471278/96114019-50c01f80-0ee5-11eb-8e11-e45c459f8494.png">

Reply:
Not sure :) should icon be visible in that case? 

![Screenshot 2020-10-15 at 20 46 52](https://user-images.githubusercontent.com/14054752/96153979-9b09c680-0f0e-11eb-9e67-9ea5602a5157.png)